HomeWork3/Maksim_Panibrat_HW3.1.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+
+int main(int argc, char* argv[]) {
+    std::cout << "This program reverses string\n";
+    const size_t kMaxSize = 31;
+    char array[kMaxSize]{};
+    std::cout << "Enter string for reverse, in range of " << kMaxSize - 1
+        << " symbols: ";
+    std::cin >> array;
+    const size_t size = strlen(array);
+    char* begin = { array }, * end = { begin + size };
```

Comment:
```suggestion
    char* begin = { array };
    char* end = { begin + size - 1 };
```
не жадничайте на строчки